docs/deployment.rst

```diff
@@ -1,63 +1,93 @@
 Deployment
 ==========
 
-moz-dev-dash is designed with `12-factor app philosophy`_ to run on `heroku`_, so you
-can easily deploy your changes to your own heroku app with `heroku toolbelt`_.
+moz-dev-dash is designed with `12-factor app philosophy`_, so you can easily
+deploy your changes to your own app.
 
 
-Deploy your own
----------------
+Deploy your own (on Mozilla Deis)
+---------------------------------
 
-#. `Create a heroku remote`_. We suggest naming it moz-dev-dash-`username`::
+To deploy the app to the Mozilla Cloud Ops Dev Deis cluster, you will need to
+`request a dev IAM account from Mozilla Cloud Ops`_.
```

Comment:
It may just be my unfamiliarity with Deis, but I wasn't sure whether I needed to complete this step. If it's not too much trouble, can we say more about who is most likely to need to do this?